frontend/companion-app/components/StartButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text, TextStyle, ViewStyle, Pressable  } from 'react-native';
+
+export function Start_Stop_Button( { state, dispatch, 
+    button_style, text_style
+     } : 
+    { state: { playing: boolean }, dispatch: Function,
+        button_style: ViewStyle, text_style: TextStyle
+     } ) {
+    
+    // const timeoutRef = useRef<Number>();
+
+    // function find_the_time() {
```

Comment:
I think the commented code can be removed. StartButton does not need to worry about the time

Reply:
Change made